Cloud runner v0 docs (#257)
game-ci/documentation: docs/cloud-runner/v0/01-introduction.md

#### Large GitHub Projects

> GitHub Actions by default run on build machines provided by GitHub. You can see info about the specifications of the build machines [Here](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners).

Reviewer: ```suggestion
GitHub Actions by default run on build machines provided by GitHub. See information about the [specifications of the build machines](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners).
```

While we're at it, instead of `[here]` it's often useful to just put the link on the relevant part (it secretly reads much easier for the end-user)
